torch/csrc/inductor/aoti_torch/generated/c_shim_generic.h

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
why are there two extra blank lines haha

Reply:
This isn't from my PR, all the rest of the c_shims are like this, so I'm leaving it as is